impeller/entity/contents/tiled_texture_contents.cc

```diff
@@ -67,12 +67,12 @@ bool TiledTextureContents::Render(const ContentContext& renderer,
   cmd.pipeline =
       renderer.GetTiledTexturePipeline(OptionsFromPassAndEntity(pass, entity));
   cmd.stencil_reference = entity.GetStencilDepth();
-  cmd.BindVertices(CreateSolidFillVertices<VS::PerVertexData>(
+  cmd.BindVertices(CreateSolidFillVertices(
       renderer.GetTessellator(),
       GetCover()
           ? PathBuilder{}.AddRect(Size(pass.GetRenderTargetSize())).TakePath()
           : GetPath(),
```

Comment:
This is one of the spots I was thinking of - this is relatively heavy to just create a set of rectangular vertices, but it's probably not a hot path anyway...

Reply:
We should not even tessellate rects. That requires some reworking. I think @bdero idea to refactor Path to Geometry to support both Vertices and Path may give us an opportunity to create a specialized Rect geometry that can skip this step.

That might make sense for circles and other shapes too.